docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Same comment here re: line 14, compute the prediction first.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Changed also</p>